### ISSUE TITLE
[a11y]: CodeSnippet multiline violations

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon#14237]
### Package

@carbon/react

### Package version

1.332

### Automated testing tool and ruleset

IBM Equal Accessibility Checker, Latest Deployment

### Description

Playwright testing do not pass for multiline codesnippet
https://github.com/carbon-design-system/carbon/actions/runs/5578503344/jobs/10192796978

<img width="922" alt="Screenshot 2023-07-18 at 9 55 44 AM" src="https://github.com/carbon-design-system/carbon/assets/2753488/11f3a5ff-6bfa-4671-b15c-75fe9695fce3">
<img width="837" alt="Screenshot 2023-07-18 at 9 56 03 AM" src="https://github.com/carbon-design-system/carbon/assets/2753488/fac4a2d3-3de0-4978-879f-b68ec14459ee">

### Reproduction/example

e2e/components/CodeSnippet/CodeSnippet-test.avt.e2e.js

### Steps to reproduce

Unskip the skipped tests in `e2e/components/CodeSnippet/CodeSnippet-test.avt.e2e.js` and run playwright test  `npx playwright test e2e/components/CodeSnippet/CodeSnippet-test.avt.e2e.js `, notice tests do not pass.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
